crates/bevy_render/src/camera/camera.rs

```diff
@@ -313,7 +313,7 @@ pub struct ExtractedCamera {
     pub physical_size: Option<UVec2>,
 }
 
-pub fn extract_cameras<M: Component + Default>(
+pub fn extract_cameras<M: Component + Sync + Default>(
```

Comment:
Now this raises an interesting discussion - in theory, there's no reason this can't support `!Sync` components, so long as they're scheduled on only one thread at a time. But how do we tell the scheduler that, without adding boilerplate somewhere (on the `Component` impl?).

This is fine for now.

Reply:
We could automatically denote any `!Sync` component as "exclusive access only" and treat `&T` as if it was `&mut T`. But that is a both a bit out of my comfort zone to do in this PR.